RFC/src/SUMMARY.md

```diff
@@ -7,6 +7,31 @@
   - [RFC-0110: Base nodes](RFC-0110_BaseNodes.md)
   - [RFC-0130: Mining](RFC-0130_Mining.md)
   - [RFC-0150: Wallets](RFC-0150_Wallets.md)
+- [RFC-0200: Tari-specific extensions to Mimblewimble](RFC-0200_BaseLayerExtensions.md)
+  - [RFC-0210: Namespace registration](RFC-0210_NamespaceRegistration.md)
+  - [RFC-0220: Asset checkpoints](RFC-0220_AssetCheckpoints.md)
+  - [RFC-0230: Asset multiparty contracts](RFC-0230_AssetContracts.md)
 - [RFC-0300: The Digital Assets Network](RFC-0300_DAN.md)
+- [RFC-0320: Validator Nodes](RFC-0320_ValidatorNodes.md)
+  - [RFC-0321: Asset discovery](RFC-0321_VNAssetDiscovery.md)
+  - [RFC-0322: Validator Node Registration](RFC-0322_VNRegistration.md)
+  - [RFC-0323: Validator Node committee selection](RFC-0323_VNCommittees.md)
```

Comment:
Maybe an additional RFC is required for VN Committee update/change protocol, that describes the process of leaving and joining pre-existing committees when a position has opened.

Reply:
Yeah, cool. We also have checkpointing ideas, so perhaps group those two together for now